chore: remove 2015 preset and add env

hozefaj opened the pull request:
Is there any requirement in setting a specific set of targets?

timdorr commented:
We should still target IE11 support. If not, I would argue that warrants a major bump.

hozefaj commented:
@timdorr So what's the recommendation, I agree that dropping ie 11 support should warrant a major version bump. 

But on the other hand, supporting ie11 also dings everyone else that use browsers that have over the air updates.

timdorr commented:
For the moment, I would include IE11. We can revisit the config for the next major release.

hozefaj commented on .babelrc:
```diff
@@ -1,30 +1,6 @@
-      ]
-    },
+  "presets": ["env"],
+  "targets": {
```
https://babeljs.io/docs/en/babel-preset-env#targets

hozefaj commented:
@timdorr I have added `ie11` as a target. Also removed the lock file. Best that a new lock file is generated before a new version of the package is published.

timdorr commented:
No, please include the lock file. That shows exactly which deps to install.

hozefaj commented:
> No, please include the lock file. That shows exactly which deps to install.

Done, added the `package.json-lock` file to the PR

timdorr commented:
Excellent. Thank you!
